### PR TITLE
osh: bump to 0.8.1

### PR DESCRIPTION
**Pull request by jedahan**
## Existing package

- [x] I am the maintainer of this package.

### CODE REVIEW

File: community/osh/build [hunk 1]
```diff
@@ -3,7 +3,7 @@
 :> nostrip
 
 ./configure \
-    --prefix=/usr
+    --prefix /usr
```
Comment (dylanaraps) [inline community/osh/build:6]: Intentional?

Comment (dylanaraps) [reply community/osh/build:6]: Usage shows `=` https://github.com/oilshell/oil/blob/master/configure#L38

Comment (jedahan) [reply community/osh/build:6]: I was following the documentation http://www.oilshell.org/release/latest/doc/INSTALL.html I will ask the maintainers of osh.

Comment (dylanaraps) [reply community/osh/build:6]: Both work. I think it's best to use `=` anyway. I don't know why both syntaxes are even supported... the usage in this PR is undocumented too.

Comment (dylanaraps) [reply community/osh/build:6]: See: https://github.com/oilshell/oil/blob/master/configure#L96-L107

**Pull request merged**